COMMITTERS.md

```diff
@@ -118,13 +118,13 @@ This section aims to provide guidelines for being a good commit citizen by
 paying attention to our automated build tools.
 
  * Don’t push on a broken build.  (A broken build is defined as a failing job
-   in [Puppet Strings](https://jenkins.puppetlabs.com/job/platform_puppet-strings_unit-ruby_master/)
+   in [Puppet Strings](https://jenkins.puppetlabs.com/job/platform_puppet-strings_unit-ruby_main/)
```

Comment:
This link looks outdated. I believe it should now be: `https://travis-ci.com/github/puppetlabs/puppet-strings`

Reply:
Updated :)